src/Stratis.SmartContracts.CLR/ContractTxData.cs

```diff
@@ -64,6 +66,9 @@ public class ContractTxData
         /// <summary>The contract code that will be executed.</summary>
         public byte[] ContractExecutionCode { get; }
 
+        /// <summary>The signatures (if any).</summary>
```

Comment:
Detail the format of what they should be pass as 👍

Reply:
Done.